### ISSUE TITLE
Can't connect anymore to HA server

### ISSUE THREAD

**Issue by PierpaoloTRV** [home-assistant/android#2147]
**Home Assistant Android version:**
2022.1.1-full

**Android version:**
Android 10

**Phone model:**
Huawey P30 (ELE-29)

**Home Assistant version:**
Home Assistant 2022.1.0.dev20211212

**Last working Home Assistant release (if known):**
Home Assistant 2022.1.0.dev20211212

**Description of problem:**
Have been using both HA and companion app for two years without problems. Since yesterday (no updates and no changes introduced) can't access anymore to HA with the companion app (can still access through web): Message saying unable to connect to HA. Tried "Waiting": no luck. Tried also uninstalling and re-installing the app: no luck. Tried also a complete fresh install (clear data, uninstall app, restart telephone): I can connect to server, as I get prompted with username and password, but after providing the correct username and password the app hangs up and exits. It does this consistently every time I try (I provide the URL, it connects and prompts for username and password, but after this the app stops working). By the way, if I provide either wrong username and password the app correctly fails the authentication, showing that server connection is established)
**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):** Unable to provide log as this would imply going past the initial onboarding

```

```

**Screenshot of problem:**

**Additional information:**

**Comment by dshokouhi** [comment]
You will need to use an app like logcat reader to pull the logs for us so we can continue with this issue.

**Comment by PierpaoloTRV** [comment]
here is the HA log:
Logger: homeassistant.components.http.ban
Source: components/http/ban.py:124
Integration: HTTP (documentation, issues)
First occurred: 9:29:14 AM (5 occurrences)
Last logged: 10:45:41 AM

Login attempt or request with invalid authentication from 37-161-14-113.mob.proxad.it (37.161.14.113). (HomeAssistant/Android ELE-L29 2022.1.1-full)

**Comment by PierpaoloTRV** [comment]
The above is the log I get if I re-install the app leaving the app data. If I clear the app data and launch the app, I am prompted with username and password (usual and correct ones), but the app hangs up with no logs recorded with logcat.
Please help

**Comment by PierpaoloTRV** [comment]
This is the log from HA:

```
Logger: homeassistant.components.http.ban
Source: components/http/ban.py:124
Integration: HTTP (documentation, issues)
First occurred: 9:29:14 AM (9 occurrences)
Last logged: 1:31:52 PM

Login attempt or request with invalid authentication from 37-161-14-113.mob.proxad.it (37.161.14.113). (HomeAssistant/Android ELE-L29 2022.1.1-full)
```

**Comment by PierpaoloTRV** [comment]
This is the logcat record:
```
[01-14 13:51:37.613 2664:3311 I/PG_ash]
packets app: io.homeassistant.companion.android

[01-14 13:51:37.613 2664:3311 I/ash_trans]
io.homeassistant.companion.android { hibernation duration=9308 Uptime=9308 } transition to: doze reason:net_packet

[01-14 13:51:37.613 2664:3311 I/PG_ash]
clear H actions!

[01-14 13:51:37.613 1656:11008 I/PGManagerService]
proxyWakeLockByPidUid, pid: 16657, uid: 10326, proxy: false

[01-14 13:51:37.613 2664:3311 I/PG_ash]
unP_wl uid = 10326 pid = 16657

[01-14 13:51:37.613 1656:11008 I/PGManagerService]
proxyWakeLockByPidUid, pid: 16889, uid: 99005, proxy: false

[01-14 13:51:37.614 2664:3311 I/PG_ash]
unP_wl uid = 99005 pid = 16889

[01-14 13:51:37.614 1013:12622 I/Pged-IPgedBinderService]
****action is 2, pid num is 3***

[01-14 13:51:37.614 1013:12622 I/Pged-Freezer]
thawPids

[01-14 13:51:37.616 2664:3311 D/PgedBinderAdapter]
unF_Z ok, pid: [16657, 16889]

[01-14 13:51:37.616 2664:3311 I/PGServer]
report state:6 event type:2 pid:0 uid:10326 pkg:io.homeassistant.companion.android to pid: 1656

[01-14 13:51:37.616 2664:3311 I/PGServer]
report state:100 event type:2 pid:0 uid:10326 pkg:16657:16889 to pid: 1656

[01-14 13:51:37.616 2664:3311 I/PG_ash]
unF_Z io.homeassistant.companion.android OK !

[01-14 13:51:37.617 1656:2364 I/PGManagerService]
proxyBroadcast:[io.homeassistant.companion.android] proxy:false

[01-14 13:51:37.617 2664:3311 I/PG_ash]
unP_BC io.homeassistant.companion.android OK !

[01-14 13:51:37.617 2664:3311 I/PG_ash]
cancel unified HB io.homeassistant.companion.android pkgs: [io.homeassistant.companion.android]

[01-14 13:51:37.617 1656:1894 I/ActivityManager]
App 10326/io.homeassistant.companion.android targets O+, restricted

[01-14 13:51:37.617 1656:2364 D/SmartHeartBeat]
setAlarmsAdjust ...

[01-14 13:51:37.617 2664:3311 I/PG_ash]
cancel unified io.homeassistant.companion.android H_B reason: left H

[01-14 13:51:37.617 1656:2028 W/HwAlarmManagerService]
removeBatchesOfPkgOrProcLocked do nothing 

[01-14 13:51:37.617 2664:3311 D/APwPowerPolicy]
gps pkg: io.homeassistant.companion.android uid:10326 duration:0

[01-14 13:51:37.617 2664:3311 I/APwPowerPolicy]
not ignore oversea frequent app gps: io.homeassistant.companion.android

[01-14 13:51:37.620 2664:3311 I/PG_ash]
unP_GPS:io.homeassistant.companion.android uid:10326 result:true

[01-14 13:51:37.621 2664:3311 I/PG_ash]
unP_GPS:io.homeassistant.companion.android uid:99005 result:true

[01-14 13:51:37.621 2664:3311 I/PG_ash]
clear H actions! unProxySensor

[01-14 13:51:37.622 2664:3311 I/powergenie_jni]
proxy sensor, uid:10326, access:0, proxy:0

[01-14 13:51:37.623 2664:3311 I/PG_ash]
UnP_Sensor:io.homeassistant.companion.android uid:10326 result:true

[01-14 13:51:37.623 2664:3311 I/powergenie_jni]
proxy sensor, uid:99005, access:0, proxy:0

[01-14 13:51:37.623 2664:3311 I/PG_ash]
UnP_Sensor:io.homeassistant.companion.android uid:99005 result:true

[01-14 13:51:37.623 2664:3311 D/APwPowerPolicy]
gps pkg: io.homeassistant.companion.android uid:10326 duration:0

[01-14 13:51:37.623 2664:3311 I/APwPowerPolicy]
not ignore oversea frequent app gps: io.homeassistant.companion.android

[01-14 13:51:37.623 2664:3311 I/PG_ash]
io.homeassistant.companion.android is quick switch pkg ,not allow P_service

[01-14 13:51:37.623 2664:3311 I/PG_ash]
io.homeassistant.companion.android is quick switch pkg ,not allow P_service

[01-14 13:51:37.623 2664:3311 I/PG_ash]
nap package: io.homeassistant.companion.android, visible: false, bgTime: 176170, scrOff: false

[01-14 13:51:37.623 16657:16657 D/SensorReceiver]
Sensor bluetooth_connection corresponding to received event android.bluetooth.device.action.ACL_DISCONNECTED is disabled, skipping sensors update

[01-14 13:51:37.619 1656:2028 I/chatty]
uid=1000(system) HeartBeatHandle identical 140 lines

[01-14 13:51:37.619 1656:2028 W/HwAlarmManagerService]
removeBatchesOfPkgOrProcLocked do nothing 

[01-14 13:51:37.624 1656:2364 V/BroadcastQueue]
Finished with ordered broadcast BroadcastRecord{7ddce21 u0 android.bluetooth.device.action.ACL_DISCONNECTED}

[01-14 13:51:37.624 1656:2364 I/ActivityManager]
App 10326/io.homeassistant.companion.android targets O+, restricted

[01-14 13:51:37.626 1656:2364 V/BroadcastQueue]
Finished with ordered broadcast BroadcastRecord{cc50246 u0 android.bluetooth.device.action.ACL_DISCONNECTED}

[01-14 13:51:37.654 2664:3150 I/PGAudioState]
receive session 33 aduio track volume state : 0

[01-14 13:51:37.654 2664:3150 I/PGAudioState]
update audiotrack volume state ,trackId 57,volume : 0.0

[01-14 13:51:37.684 1656:1656 I/HwNotificationService]
specialType is:floating_window_notification ,topFullscreen is:false,isRequestSingleLine :false,HWMULTIWIN_ENABLED is:true

[01-14 13:51:37.684 1656:1656 D/HwNotificationService]
mGameDndStatus is:false ,isGameDndSwitchOn is:false,isDeferLaunchActivity is:false

[01-14 13:51:37.685 2664:3150 I/AppManager]
app:com.dp.logcatapp Notification update time:2904315

[01-14 13:51:37.685 2664:3150 I/AppManager]
app : com.dp.logcatapp add Notification persist:true now Ids:[1]

[01-14 13:51:37.686 2226:2226 I/NotificationListener]
onNotificationPosted: StatusBarNotification(pkg=com.dp.logcatapp user=UserHandle{0} id=1 tag=null key=0|com.dp.logcatapp|1|null|10377: Notification(channel=logcat_channel_01 pri=1 contentView=null vibrate=null sound=null tick defaults=0x0 flags=0x62 color=0xff009688 category=service actions=2 vis=PRIVATE)) important=2, post=1642164697581, when=1642164697579,showWhen=true, vis=0, userid=0, hasFullScreenIntent=false

[01-14 13:51:37.689 2226:2226 I/NotificationViewHierarchyManager]
updateNotificationViews activeNotificationSize=1

[01-14 13:51:37.690 2226:3772 I/HwNotificationEx]
preActionButton

[01-14 13:51:37.691 2226:3772 I/HwNotificationEx]
preActionButton

[01-14 13:51:37.691 2226:2226 I/InterruptionStateProvider]
No heads up: unimportant notification: 0|com.dp.logcatapp|1|null|10377

[01-14 13:51:37.692 2226:2350 E/HiEvent.Payload]
new JsonPayload exception:Unterminated object at character 88 of {pkg:com.dp.logcatapp,id:1,tag:"null",channelid:"logcat_channel_01",channelname:Logcat ser,isOnGoing:"true",importance:2}

[01-14 13:51:37.693 2226:2350 I/HiView.HiEvent]
trace is not open or not valid

```

**Comment by PierpaoloTRV** [comment]
Problem solved!
I tried all sort of things, and I cannot therefore offer a final solution to anyone else who may experience the same issue. I can only say it was not (or should not have been) attributable to the HA Server, as I tried also a mirror hosted in a different computer; it was not a problem of authentication, as I was able to connect with my mobile to HA through the browser (Chrome and Brave).
Once I came home, the last attempt I made was to set the internal URL under WiFi and then cycle back to the external URL using mobile data, after resetting all containers in my RPi (only "caddy" could have played a role as the problem could not lie with HA container: as I stated before I also made an attempt with a different HA server hosted in a different PC of my home network).
